_posts/2018-03-23-crossword-data-format.md

```diff
@@ -0,0 +1,116 @@
+
+```yaml
+version: standard v1
+name: Crossword 15800
+author: Julius
+editor: Colin Inman
+copyright: 2018, Financial Times
+publisher: Financial Times
+pubdate: 2018/03/07
+size: 15x15
+across:
+- (1,1) 1. Reviewed bistrot/cafe <i>La Maison</i>: sandwiches "criminal" (10)
+- (12,1) 6. ,10. Steer clear of wildcat general strike; it's a tragedy eerst (4)
+- (1,3) 9. New salary caps not introduced at work; it gets up my nose (5,5)
+- (12,3) 10. See 6 across (4)
+- (4,5) 12. Live like a king with dad in castles in far-flung places (7,5)
+- (1,7) 15. Got up and prepared the pizza base (6,3)
+- (11,7) 17. She's an inspiration for occasionally weird author (5)
+- (1,9) 18. Heads of Foreign Office committee unusually struggle to concentrate . . . . (5)
+- (7,9) 19. . . . . as multiple channels stream Tory with excellent oratory skills (9)
+- (1,11) 20. Design squalid sites as means to raise quick cash? (6,6)
+- (1,13) 24. ,26. Suites arranged to suit Norwegian's booking? (4)
+- (6,13) 25. Like the tale of bird fencing Romeo's mate's stolen car? (10)
+- (1,15) 26. See 24 across (4)
+- (6,15) 27. Fake uni degrees heavily attacked (5,5)
+down:
+- (1,1) 1. ,2. Somewhat hormonal, Isabella is an enigmatic figure (4)
+- (3,1) 2. See 1 down (4)
+- (5,1) 3. Didn't Trump go after journalist with court case? (8,4)
+- (7,1) 4. Old Bill gathered wood (5)
+- (9,1) 5. Hirsute character in Oxford University called Brown (9)
+- (13,1) 7. In European empty bar I had dinner date resembling Edna (10)
+- (15,1) 8. Rustic logs chopped for those making camp fire? (4,6)
+- (11,4) 11. Names US mountains as stated by Yankee (12)
+- (1,6) 13. Let rip guff; foul smell gets up one's nose! (7-3)
+- (3,6) 14. Piece of ebony ruler? (5,5)
+- (7,7) 16. Not for the first time, falls over, leaving artist to support old morthern church (4,5)
+- (9,11) 21. Composer putting note in envelope (no stamp required) (5)
+- (13,12) 22. ,23. Book a month to take over from glum donkey (4)
+- (15,12) 23. See 22 down (4)
+```
+The above example of the new crossword data format describes [Crossword No.15,800 Set by Julius on 2018/03/07](https://www.ft.com/__origami/service/image/v2/images/raw/ftcms:bdb7ea5a-1875-11e8-9e9c-25c814761640?source=crosswordsftcom),
+
+{% include figure-noimgservice.html src="https://www.ft.com/__origami/service/image/v2/images/raw/ftcms:bdb7ea5a-1875-11e8-9e9c-25c814761640?source=crosswordsftcom" alt="A example of the new crossword data format" width="100%" %}
+
+and drives [an interactive version](https://labs.ft.com/crosswords/2018/03/07/Puzzle-15800-dynamic.html).
+
+# The new format
```

Comment:
Section titles should be `<h2>` not `<h1>`

Reply:
done